pkg/datapath/linux/config/config.go

```diff
@@ -1073,6 +1074,11 @@ func (h *HeaderfileWriter) WriteEndpointConfig(w io.Writer, e datapath.EndpointC
 	return h.writeTemplateConfig(fw, e)
 }
 
+func enforceFibLookup(e datapath.EndpointConfiguration) bool {
```

Comment:
Could you put this on the type of `option.Config` instead? Then we wouldn't add another dependency on the global variable.